src/pages/learn/schema.mdx

```diff
@@ -101,7 +92,7 @@ type Query {
 
 Mutations work in a similar way - you define fields on the `Mutation` type, and those are available as the root mutation fields you can call in your query.
 
-It's important to remember that other than the special status of being the "entry point" into the schema, the `Query` and `Mutation` types are the same as any other GraphQL object type, and their fields work exactly the same way.
+It's important to remember that other than the special status of being the entry points, the `Query` and `Mutation` types are the same as any other GraphQL object type, and their fields work exactly the same way.
```

Comment:
Maybe it's best to reintroduce the `schema` keyword here:

````suggestion
It's important to remember that other than the special status of being the entry points, the `Query` and `Mutation` types are the same as any other GraphQL object type, and their fields work exactly the same way.

You can choose to name your entry point types differently; if you choose to do so then you will need to inform GraphQL of the new names using the `schema` keyword:

```graphql

```
schema {
  query: MyQueryType
  mutation: MyMutationType
}
```
````

Reply:
What percent of users would you estimate rename their entry points?

Reply:
What percent of users would you estimate rename their entry points?

Reply:
Hard to say; but our very own `swapi-graphql` example API does: https://graphql.org/swapi-graphql/?query=%7B__typename%7D